attributes/default.rb

```diff
@@ -1,7 +1,9 @@
 # encoding: utf-8
 
-# This attributes is only taken into account for RHEL installations
-default['aurora']['version'] = '0.12.0-1'
+default['aurora']['version'] = case node['platform_family']
+    when 'rhel' then '0.12.0-1'
+    else ''
+end
```

Comment:
I removed the comment because I found the block explicit enough (though I may be wrong). We have to specify the version we want on Centos (because we fetch the RPMs directly), but we can take the latest available package on debian. Do you think I should add this as a comment ?